README.md

```diff
@@ -294,7 +297,8 @@ app = App(routes=[
 
 ### Reversing URLS
 
-Rather than build URLs by hand, it is possible to generate URLs based on the endpoint, using `reverse_url()`.
+Rather than build URLs by hand, it is possible to generate URLs based on the
+endpoint, using `reverse_url()`.
```

Comment:
In markdown, you shouldn't insert your own linebreaks, leave paragraphs all on the same line